azavea/cac-tripplanner
# Feature/kak/isochrone filter#876

## Overview

Limit destinations in the 'explore' tab sidebar to those within the displayed isochrone. 
Will display all destinations if origin field is cleared.


### Demo

![image](https://user-images.githubusercontent.com/960264/31361439-5ac830cc-ad21-11e7-91a1-c2894c49d6cf.png)


### Notes

Also removed the `prototypes` directory no longer in use.

## Testing Instructions

 * `vagrant ssh app`
 * `cd /opt/app/src && npm run gulp-development`
 * Go to explore mode with 'map view' button on home page or toggle on bottom of directions view
 * Only destinations within isochrone should show
 * Clear origin field and all destinations should show

Closes #876
